assemblies/pom.xml

```diff
@@ -14,6 +14,7 @@
   <description>Pentaho Community Plugin Framework assemblies</description>
 
   <modules>
-    <module>cpf-plugin-pdi</module>
+    <module>cpf</module>
```

Comment:
Why rename the folder if the feature name is `pentaho-cpf-plugin-pdi`? (prefix pentaho seems to be mandatory)

Reply:
Nice catch, I forgot to change the feature name when renaming the folder